doc/qutebrowser.1.asciidoc

```diff
@@ -34,7 +34,7 @@ show it.
 *'URL'*::
     URLs to open on startup (empty as a window separator).
 
-=== optional arguments
+=== options
```

Comment:
No idea why this changed, let's see if the CI is happy with it. Maybe different Python versions.